.github/workflows/cd-development.yml

```diff
@@ -0,0 +1,66 @@
+name: Continuous Delivery
```

Comment:
We either should remove development from file name or add it to name and job name